I think the test failure is related:
```console
FAILED tests/components/mysensors/test_text.py::test_text_node - AssertionError: assert 'Value 123456...um length 25)' == 'Value 123456...um length 25)'
  - Value 12345678123456781234567812 for Text Node 1 1 is too long (maximum length 25)
  ?                                      ^   ^^   ^ ^
  + Value 12345678123456781234567812 for text.text_node_1_1 is too long (maximum length 25)
  ?                                      ^   ^^^^^^^   ^ ^
```

--- a/tests/components/mysensors/test_text.py
+++ b/tests/components/mysensors/test_text.py
@@ -61,5 +61,5 @@ async def test_text_node(
         )
 
     assert str(err.value) == (
-        f"Value {value} for Text Node 1 1 is too long (maximum length 25)"
+        f"Value {value} for text.text_node_1_1 is too long (maximum length 25)"
     )

